deps/exokit-bindings/glfw/src/glfw.cc

```diff
@@ -1349,10 +1349,15 @@ NAN_METHOD(GetClipboard) {
   }
 }
 
+
 NAN_METHOD(SetClipboard) {
-  NATIVEwindow *window = (NATIVEwindow *)arrayToPointer(Local<Array>::Cast(info[0]));
-  String::Utf8Value str(info[0]->ToString());
-  glfwSetClipboardString(window, *str);
+  NATIVEwindow *window = GetCurrentWindowContext();
+  if (info.Length() == 1) {
+    Nan::Utf8String utf8_value(info[0]);
+    glfwSetClipboardString(window, *utf8_value);   
+  } else {
+      return Nan::ThrowTypeError("must provide arg length > 0");
```

Comment:
This error is not in line with the if. Can we just say “invalid arguments”? The stack will point here for anyone who cares to debug.

Reply:
Done!